src/Test.lsp

```diff
@@ -22,7 +22,7 @@
  | *ALU:testSuites* - assoc list of test suites and a list of their tests
  | *ALU:testsRun* - counter for total tests run in the current batch
  | *ALU:failMessages* - list of messages to print after testing
- | *ALU:startTime* - date number for start of testing
+ | *ALU:startTime* - milliseconds since boot time for start of testing
```

Comment:
I can't test these timing change now, but this seems like a good approach. Nice work!